scripts/check_compliance.py

```diff
@@ -165,6 +167,23 @@ def run(self):
         # single Kconfig tree and warns for all references to undefined symbols
         os.environ["KCONFIG_STRICT"] = "y"
 
+        kconfig_env_var_list_path = os.path.join(self.zephyr_base,
+                                    os.path.normpath("cmake/kconfig_env_vars.txt"))
```

Comment:
Can drop the `os.path.normpath()`. Will be a no-op for `"cmake/kconfig_env_vars.txt"`.

Reply:
Yeah that's what the idea was but I think `os.path.join` will fix it anyways so I guess i'll remove it

Reply:
fixed